### PR TITLE
Rename tiles.json to tileset.json

### LINKED ISSUE #49: Rename tiles.json to tileset.json
This is probably the right thing to do since it is more precise.

### PR DESCRIPTION
**Pull request by pjcozzi**
Merge #52 first.

Fixes #49.

@TomPed can you make the corresponding code change in Cesium?  Make sure to rename the `tiles.json` files used in the tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Contents:
 * [Spec status](#spec-status)
 * [Introduction](#introduction)
 * [Tile metadata](#tile-metadata)
-* [tiles.json](#tilesjson)
+* [tileset.json](#tilesjson)
```
Comment (lilleyse) [inline README.md]: Should `#tilesjson` be `#tileset.json`?

File: README.md [hunk 2]
```diff
@@ -301,7 +301,7 @@ A tileset can contain any combination of tile formats.  3D Tiles may also suppor
    * [Will 3D Tiles replace KML?](#will-3d-tiles-replace-kml)
 * [Technical Q&A](#technical-qa)
    * [How do 3D Tiles support heterogeneous datasets?](#how-do-3d-tiles-support-heterogeneous-datasets)
-   * [Will tiles.json be part of the final 3D Tiles spec?](#will-tilesjson-be-part-of-the-final-3d-tiles-spec)
+   * [Will tileset.json be part of the final 3D Tiles spec?](#will-tilesjson-be-part-of-the-final-3d-tiles-spec)
```
Comment (lilleyse) [inline README.md]: Also here: `#Will-tileset.json-be-part-of-the-final-3D-Tiles-spec`

**Comment by TomPed** [comment]
How would I know which `README.md` file these changes are to be made?

**Comment by lilleyse** [comment]
The `README.md` changes I mentioned are for this repo only (the 3D tiles spec), so you don't have to worry about them.

**Comment by TomPed** [comment]
Got it.

When I search for `tiles.json` a lot of files show up am I to change them all?

**Comment by lilleyse** [comment]
Yeah you can change them all.

**Comment by TomPed** [comment]
I will also change the file `tiles2.json` to `tileset2.json` and `tiles3.json` to `tileset3.json`

File: README.md (hunk 1 above)
Comment (pjcozzi) [inline README.md]: Yes.

**Comment by pjcozzi** [comment]
Updated links.

**Comment by lilleyse** [comment]
Merged.

**Pull request merged**